template/contracts/src/BuyMeACoffee.sol

```diff
@@ -25,7 +25,7 @@ pragma solidity 0.8.23;
  * @dev Memo struct
  */
 struct Memo {
-    uint numCoffees;
+    uint256 numCoffees;
```

Comment:
Out of curiosity, what this change?

Reply:
`uint` is [an alias](https://docs.soliditylang.org/en/v0.8.23/types.html#integers) for `uint256`. In general, using the more explicit `uint256` is preferred, and then that should be used consistently across the contract.

Reply:
This was done by the "forge fmt" command.. we should ensure that is being run on checkin.